python/src/trezorlib/messages.py

```
@@ -10693,14 +10706,17 @@
     FIELDS = {
         1: protobuf.Field("credentials", "WebAuthnCredential", repeated=True, required=False, default=None),
+        100: protobuf.Field("id_map", "bytes", repeated=True, required=False, default=None),
     }
 
     def __init__(
         self,
         *,
         credentials: Optional[Sequence["WebAuthnCredential"]] = None,
+        id_map: Optional[Sequence["bytes"]] = None,
     ) -> None:
         self.credentials: Sequence["WebAuthnCredential"] = credentials if credentials is not None else []
```

Ruff (0.8.2)

Remove quotes from type annotation

Remove quotes

(UP037)